src/fmu/ensemble/realizationcombination.py

```diff
@@ -117,16 +120,36 @@ def get_df(self, localpath):
             return result.reset_index()
         return result.dropna().to_dict()
 
-    def to_virtual(self):
+    def to_virtual(self, keyfilter=None):
         """Evaluate the current linear combination and return as
         a virtualrealizatione.
+
+        Args:
+            keyfilter (list or str): If supplied, only keys matching wildcards
+                in this argument will be included. Use this for speed reasons
+                when only some data is needed. Default is to include everything.
+                If you supply "unsmry", it will match everthing key that
```

Comment:
Same as in ensemblecombination.py
```suggestion
                If you supply "unsmry", it will match every key that
```